invenio_administration/templates/semantic-ui/invenio_administration/header/header_login.html

```diff
@@ -24,7 +24,24 @@
         </form>
     {%- else %}
 
+        <div role="menuitem" class="custom-plus-menu ui dropdown floating pr-15 computer only">
+            <i class="fitted plus icon inverted"></i>
+            <i class="fitted dropdown icon inverted"></i>
+            <div class="menu">
+                    <a class="item"
+                        style="color:black;"
+                        href="/oai-pmh/create">{{ _('New OAI-PMH set') }}
```

Comment:
please use url_for function instead of hardcoding the url

Reply:
I hardcoded this value because I didn't see it defined anywhere. I only found the URL in https://github.com/inveniosoftware/invenio-rdm-records/pull/1044/files, and it was there hardcoded too. Should I define the write in the configuration of invenio-rdm-records, then define it in the blueprint invenio_rdm_records_ext, so I can use it with url_for?